app/src/main/java/com/beemdevelopment/aegis/ui/MainActivity.java

```diff
@@ -768,7 +777,7 @@ public void onLocked(boolean userInitiated) {
         if (userInitiated) {
             startAuthActivity(true);
         } else {
-            super.onLocked(userInitiated);
+            super.onLocked(false);
```

Comment:
Why was this changed?

Reply:
Because it always is false there , Because if userInitiated is false it calls that super and it is not related to this PR, Optional

Reply:
You're right. This is fine for now, but I'd prefer it if you could separate any unrelated changes in the future.